src/styles/cardStyles.ts

```diff
@@ -19,7 +19,7 @@ export const CARD_STYLE: CSSProperties = {
   textAlign: "center",
   textSizeAdjust: "100%",
   lineHeight: "25.6px",
-  textDecoration: "none"
+  textDecoration: "none",
```

Comment:
Omit the comma of the last item of an object.

Reply:
Done.